__tests__/identity.js

```diff
@@ -21,6 +21,10 @@ describe('Identity', () => {
         global.URL = u;
     });
 
+    beforeEach(() => {
+        window.sessionStorage.clear();
+    });
```

Comment:
It was necessary, since by default both `Identity` and `Monetization` was using session storage from global window object for cache, which was shared between tests, and resulted in inconsistent results, depending on tests order.

Reply:
Those issues could be resolved by explicite passing e.g. `window: {}` to each constructor call, but I thought that this solution is more convenient.